support/Sources/GraphQL.swift

```diff
@@ -260,6 +260,14 @@ public struct GraphQLResponse<DataType: GraphQL.AbstractResponse> {
 	}
 }
 
+public protocol GraphQLQuery {
+	associatedtype Response
```

Comment:
I would like to restrict this to a more specific interface at some point (basically to what AbstractResponse defines) so we can do more with this type